go.mod

```diff
@@ -1,18 +1,18 @@
 module github.com/konveyor/openshift-velero-plugin
 
-go 1.19
+go 1.22.0
```

Comment:
Should we make this 1.22.2 to match the other repos?

Reply:
I had asked him to match https://github.com/openshift/velero/blob/konveyor-dev/go.mod so.. 1.22.0 is what we have atm.

Reply:
not opposed to 1.22.2 tho.

Reply:
Oh -- good point. upstream main is still on 1.22.0 -- that being said, I don't imagine 1.15 will be on anything earlier than 1.22.2. So .0 is fine here, but we'll need to remember to update it when we branch for oadp-1.5, or we can switch to .2 now since that's what we have in 1.4 both upstream and downstream.

Reply:
we can do that for release branch perhaps ?

Don't we get 1.22.2 automatically via rebase?

Reply:
Currently main has 1.22.0 and release-1.14 has 1.22.2.

Reply:
interesting.